src/entry/aws-console.ts

```diff
@@ -231,7 +231,7 @@ async function init(): Promise<AwsConsole> {
   return aws;
 }
 
-if (window.location.href.includes('console.aws.amazon.com/console/home')) {
+if (window.location.href.includes('console.aws.amazon.com/')) {
```

Comment:
Just curious about this, as it seems redundant when manifest is already doing the same check?

Reply:
IIRC - The manifest syntax has some limits and allows the scripts to run on any console path/page. I was previously running into some odd loop behavior and put this check in to prevent the scripts from running on unexpected console pages prior to hitting the actual console homepage.

This can likely be removed.

There's a couple other non-matching situations I'd like to fix-up as well (gov cloud, .CN, etc.). I'll take a look sometime this in the next couple days.